composer.json

```diff
@@ -74,6 +74,9 @@
         "allow-plugins": {
             "composer/package-versions-deprecated": true,
             "php-http/discovery": true
+        },
+        "platform": {
```

Comment:
this is great! I never knew about this

https://getcomposer.org/doc/06-config.md#platform